### ISSUE TITLE
Add a tipping option to the vote transaction

### ISSUE THREAD

**Issue by skzap** [dtube/avalon#53]
Since we launched the chain for real, and coins have (seemingly increasing) value, it became hard for minnows to earn their few first DTCs with curation rewards.

We already have a tipping button that does the same as a normal transfer, but we all know that giving away money for free is psychologically difficult in the world we live in.

The way I see it is, I wouldn't mind donating my curation rewards to the author of the video sometimes. I see a lot of legit minnows posting contents with <100 VP, and when I vote it with my @heimindanger account, I give them 0.01 at best, while printing a few whole coins to myself... Somehow I wouldn't mind my curation rewards going to the author when I do that.

I believe we could add a `isTip` boolean option to the VOTE transaction. The only difference for those votes, is that instead of being claimable by the voter, it would be claimable by the author of the video. Implementation in the chain should be pretty straightforward.

Please let me know if you see any fallacy with such an option, otherwise I think it would be a good improvement to fix the issue of the legit minnow accounts struggling to earn coins.

**Comment by skzap** [comment]
P.S.: Such a change will obviously require a hard-fork.

**Comment by tibfox** [comment]
That sounds extremely amazing and I am in 100% for that

**Comment by nannal** [comment]
As opposed to a bool could this be a float so I provide a percentage of the rewards to the author?

I understand there would need to be some rounding

**Comment by Birdinc** [comment]
This sounds like a good way to support minnows and grow the platform.

**Comment by hightouch67** [comment]
As we already discussed this would be a great addition, not only to help new users to get onboarded with a tipvote but also to improve/add an option to the avalon curation meaning.

@nannal I like the idea (pretty useful for big curators) but dunno if most of users want to have that extra step. The idea is to give a tipvote not to share the curation. If we go that way is not a tipvote anymore but a *curation share"? Not that those are totally different but for me they have different meanings.
Also technically it makes it a bit weird (it's still possible to implement it) but not clean as the simple tipvote.

**Comment by d00k13dotcom** [comment]
I really like this idea, I see myself gaining massive rewards when trying to motivate new creators. Often I am having to either follow up with people in explaining how the system works and even tipping to get them started. I have no problem doing this but I believe by making it a fluid transition from the rewards rather then the pocket of a curator we will likely see more interest.

Imagine the button below the slider has a check box much like the how we can add tags. "Send As Tip" with a confirmation second vote slider that pops up selecting % to donate. Then a confirmation window when trying to vote "I agree to sacrificing said % of my curation rewards to the author".

Simply put, having it right there in some way as a reminder each time I vote will make me think about possibly giving it as a tip rather then not and most likely providing liquidity to our economy. I am guilty for not even noticing the tip button at first as I am so singly focused on the vote itself.

**Comment by techcoderx** [comment]
Author tip hardfork has been approved and set to activate at block 4,800,000
